api/ReadableStream.json

```
@@ -95,14 +95,15 @@
       "async_iterable": {
         "__compat": {
           "description": "Async iterable (<code>@@asyncIterator</code>)</code>",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/ReadableStream#async_iteration",
           "spec_url": "https://streams.spec.whatwg.org/#rs-asynciterator",
           "support": {
             "chrome": {
               "version_added": false
             },
             "chrome_android": "mirror",
             "deno": {
-              "version_added": false
+              "version_added": true
```

Thanks @lucacasonato!

```suggestion
              "version_added": "1.0"
```